docs/source/test_network.md

```diff
@@ -303,33 +299,31 @@ export CORE_PEER_ADDRESS=localhost:7051
 The `CORE_PEER_TLS_ROOTCERT_FILE` and `CORE_PEER_MSPCONFIGPATH` environment
 variables point to the Org1 crypto material in the `organizations` folder.
 
-If you used `./network.sh deployCC` to install and start the fabcar chaincode,
+If you used `./network.sh deployCC` to install and start the asset-transfer (basic) chaincode,
 you can now query the ledger from your CLI. Run the following command to get the
-list of cars that were added to your channel ledger:
+list of assets that were added to your channel ledger:
 ```
```

Comment:
The assets are not on the ledger in this flow. This could be an opportunity to invoke the smart contract first, followed by a query.

Reply:
Done.